integration/test/basic/main_test.go

```diff
@@ -8,36 +8,25 @@ import (
 	"os"
 	"testing"
 
-	"github.com/giantswarm/appcatalog"
-	e2esetup "github.com/giantswarm/e2esetup/v2/chart"
-	"github.com/giantswarm/e2esetup/v2/chart/env"
-	"github.com/giantswarm/e2etests/v2/basicapp"
-	"github.com/giantswarm/helmclient/v2/pkg/helmclient"
-	"github.com/giantswarm/k8sclient/v4/pkg/k8sclient"
+	"github.com/giantswarm/apptest"
 	"github.com/giantswarm/micrologger"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
-	"github.com/giantswarm/cluster-autoscaler-app/integration/templates"
+	"github.com/giantswarm/cluster-autoscaler-app/integration/env"
 )
 
 const (
-	app            = "cluster-autoscaler"
-	appName        = "cluster-autoscaler-app"
-	catalogURL     = "https://giantswarm.github.io/default-catalog"
-	testCatalogURL = "https://giantswarm.github.io/default-test-catalog"
+	app         = "cluster-autoscaler"
```

Comment:
I think this is not used

```suggestion
```

Reply:
The naming is a bit of a pain but It's needed because the deployment name doesn't have the -app suffix.